x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/feature_access/components/agent_policy_settings_complete.cy.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loadPage } from '../../../../tasks/common';
+import { login } from '../../../../tasks/login';
+import { createAgentPolicyTask, getEndpointIntegrationVersion } from '../../../../tasks/fleet';
+import type { IndexedFleetEndpointPolicyResponse } from '../../../../../../../common/endpoint/data_loaders/index_fleet_endpoint_policy';
+import type { PolicyData } from '../../../../../../../common/endpoint/types';
+
+const getFleetAgentPolicySettingsUrl = (policyId: string) =>
```

Comment:
I think fleet exposes services out of `common/` that enables you to get the UI paths - please use that instead of hardcoding it here.

Also: (suggestion) create a reusable function (ex. `navigateToFleetAgentSettings()`) in `screens/fleet/agent_settings.ts` (new file) and then use that. You may also want to create some functions (if appropriate) for working with the components in that screen instead of having your code hardcode the test subj ids in multiple places.

Reply:
No UI paths in `common`, only API. I did however moved components related tasks to `screens`, thanks ;)